site/content/contribute/more-info/mobile/developer-setup/_index.md

```diff
@@ -8,207 +8,250 @@ aliases:
   - /contribute/mobile/developer-setup
 ---
 
-The following instructions apply to the mobile apps for iOS and Android built in React Native. Download the iOS version {{< newtabref href="https://apps.apple.com/us/app/mattermost/id1257222717" title="here" >}} and the Android version {{< newtabref href="https://play.google.com/store/apps/details?id=com.mattermost.rn" title="here" >}}. Source code can be found at the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="GitHub Mattermost Mobile app repository" >}}.
+The following instructions apply to the mobile apps for iOS and Android built in React Native.
+Download the iOS version {{< newtabref href="https://apps.apple.com/us/app/mattermost/id1257222717" title="here" >}} and the Android version {{< newtabref href="https://play.google.com/store/apps/details?id=com.mattermost.rn" title="here" >}}.
+Source code can be found at the {{< newtabref href="https://github.com/mattermost/mattermost-mobile" title="GitHub Mattermost Mobile app repository" >}}.
 
-If you run into any issues getting your environment set up, check the {{< newtabref href="https://docs.mattermost.com/deploy/mobile-troubleshoot.html" title="Troubleshooting" >}} section in the left sidebar for common solutions.
-
-{{<note "Note:">}}
-**This guide describes how to set up the development environment on macOS or Linux.**
-{{</note>}}
+If you run into any issues getting your environment set up, check the {{< newtabref href="https://docs.mattermost.com/deploy/mobile-troubleshoot.html" title="Troubleshooting" >}} section of the product docs for common solutions.
 
+{{<note "iOS mobile app">}}
 A macOS computer is required to build the Mattermost iOS mobile app.
+{{</note>}}
 
 ## Environment setup
 
-### iOS and Android
+The following instructions apply to both iOS and Android mobile apps.
+On macOS, we recommend using {{< newtabref href="https://brew.sh" title="Homebrew" >}} as a package manager.
 
-Install the following prerequisite software to develop and build the iOS or Android apps. For macOS, we recommend using {{< newtabref href="https://brew.sh" title="Homebrew" >}} as a package manager.
+### Install Cygwin (Windows only)
 
-#### Node and NPM
-We recommend using npm version 7 (with either Node 15 or 16). To make switching easier, many of our team use {{< newtabref href="https://github.com/nvm-sh/nvm" title="nvm" >}} to manage the npm and node versions.
-
-#### Install Cygwin (Windows only)
-##### Windows 10
 - {{< newtabref href="https://cygwin.com/faq/faq.html#faq.what.supported" title="Which versions of Windows does Cygwin support" >}}?
 - Install Cygwin from {{< newtabref href="https://www.cygwin.com/" title="here" >}}.
-- Make sure to install `make` and `patch` while installing Cygwin.
+- Make sure to install the `make` and `patch` packages while installing Cygwin.
```

Comment:
I'm not sure this is required anymore, have anyone tested the setup on Windows ?

Reply:
@neflyte - Is testing the setup on Windows something you're open to helping with?

Reply:
Yes; I'll test the setup on Windows as far as I can. :)